tools/plugin-base/src/main/java/io/spine/tools/gradle/project/Dependant.java

```diff
@@ -53,6 +53,24 @@
      */
     void exclude(Dependency dependency);
 
+    /**
+     * Forces all project configurations to fetch the particular dependency version.
+     *
+     * @param dependency
+     *         the forced dependency
+     * @param version
+     *         the required version of the dependency
+     */
+    void force(Dependency dependency, String version);
```

Comment:
We have a composite type for a dependency with a version — `Artifact`. Please use it as the single param type instead.